tests/attacks/lucash-flap-end.random.mcd.expected

```diff
@@ -39,7 +39,7 @@
             .Map
           </cat-ilks>
           <cat-live>
-            false
+            true
```

Comment:
I'm a bit perplexed by this and the other changes to the `live` flags...since this attack specifically requires emergency shutdown to have been activated, I'd think that `false` would be the right thing to have for `live` cell values.

Reply:
Hmmm, that's true that the attack sequence requires the shutdown to happen to make sense.

I shortened the attack sequences to the exact prefix where the `Violated` shows up in the `<properties>` cell, for the purpose of knowing how long to run the random-tester for. The actual property being checked is that "No non-auth Flap kicks are set off", which is triggered before the shutdown happens (specifically, when the non-authed flap-kick happens).

This arguably isn't a very useful property, so perhaps we should come up with a better property which encompasses this attack and the "No non-auth Flip kicks are set off" attack.

For now, I'll add an item to the meeting notes for us to brainstorm better/more general properties for catching these attacks, and I'll extend these attack sequences again to exhibit the full attack (not just tho non-authed flip/flap kicks).

Reply:
Taking a first stab (in plain English) at the Flip one:
The attack occurs when the end process attempts to reimburse a user for their bid when that user has not submitted a bid before.

So an exception has occurred if there are a sequence of transactions that includes a:
```
yank(uint)  -> vat.move(END, guy, bid)
```
without a preceding:
```
vat.move(guy, gal, bid)
```

Reply:
similarly, we can say the flap bug occurred when there is a:
```
yank(uint) -> gem.move(this, guy, bid)
```

without a preceeding:
```
gem.move(guy, this, bid)
```

Reply:
Hmmm, these are more general properties, and would catch these particular bugs. I was hoping we could find some measurable we can track though which produces the same bugs, rather than looking for specific sequences of allowed/disallowed actions. Statements about measurables will generalize better to other situations.

Reply:
I think that the flip attack might be detectable with eth TotalDebtBounded property, or perhasp the Debt == Sum(Dai) property. But I'm not sure.

Reply:
The approach Chris lays out will work, but I feel like there should be something more fundamental...something that can be deduced just from looking at the overall dai balances and the bid values of all auctions...i.e. you ought be able to detect it from a snapshot of the various account values post-attack, with no knowledge of the history. I can't quite put my finger on it yet, though...

Reply:
One simple idea: if `End.debt  != 0 && End.debt < Vat.debt`, that indicates that debt was generated post-`thaw`, which should not happen, and would detect at least the `flip` attack (I haven't thought about the `flap` attack at all yet).

Reply:
Well, we already detect if `End.debt` goes up post-`cage` (with `totalDebtBounded` property), so I think we should see if we can get it to find the flip attack using that property.

Reply:
On further thought, the property I propose above is indeed too aggressive...e.g. if `skip` gets called post-`thaw` (even on a real auction), `Vat.debt` will increase. This is arguably a flaw in the system itself though!